PyU4V/tests/unit_tests/pyu4v_common_data.py

```diff
@@ -239,15 +277,15 @@ class CommonData(object):
     compr_report = {'storageGroupCompressibility': [
         {'num_of_volumes': 6, 'storageGroupId': storagegroup_name,
          'allocated_cap_gb': 0.0, 'used_cap_gb': 0.0,
-         'compression_enabled': 'false'}, ]}
+         'compression_enabled': 'False'}, ]}
 
     volume_details = [{'cap_gb': 2,
                        'num_of_storage_groups': 1,
                        'volumeId': device_id,
                        'volume_identifier': 'my-vol',
                        'wwn': volume_wwn,
-                       'snapvx_target': 'false',
-                       'snapvx_source': 'false',
+                       'snapvx_target': 'False',
+                       'snapvx_source': 'False',
```

Comment:
These should not have changed, our REST API expects lower case string 'false' and 'true', same as settings above, if you are breaking tests then you need to resolve the function and not change the common data to suit the new functions